.env.example

```diff
@@ -13,4 +13,8 @@ DB_NAME=
 DB_USER=
 DB_PASSWORD=
 DB_HOST=
-DB_DIALECT=
\ No newline at end of file
+DB_DIALECT=
+
+
+# brcypt
+SALT_ROUNDS=10
```

Comment:
@zaheerbabarkhan where is it being used?

Reply:
i have not connected it to config, will do that